backend/EasyMeets.Core/EasyMeets.Core.Common/DTO/Availability/Schedule/ScheduleDto.cs

```diff
@@ -2,7 +2,8 @@
 
 public class ScheduleDto
 {
-    public int TimeZone { get; set; }
+    public string TimeZoneValue { get; set; } = string.Empty;
+    public string TimeZoneName { get; set; } = string.Empty;
```

Comment:
Why not a separate entity?

Reply:
Because we have only one dto for all stuff. I could use separate entity if we had more dtos for other operations